include/m/m_mtx.h

```diff
@@ -64,10 +64,10 @@ class mMtx_c : public EGG::Matrix34f {
     void rot(int, int); // does some werrd operation to rotate the matrix
     bool quatRelated();
 
-    void trans(const mVec3_c &v) {
+    void transS(const mVec3_c &v) {
         PSMTXTrans(*this, v.x, v.y, v.z);
     }
-    void trans(f32 x, f32 y, f32 z) {
+    void transS(f32 x, f32 y, f32 z) {
```

Comment:
Going with the existing pattern of an `S` suffix for functions that Set the matrix and `M` for functions that Modify/Mutate the matrix